### PR TITLE
Fixes for PT-2825 and PT-2829

### PR DESCRIPTION
**Pull request by allasm**

### CODE REVIEW

**Comment by allasm** [comment]
@sdumitriu @veronikaslc please review

File: components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ define([
                                    "Myeloma" : "Myeloma",
                                    "Leukemia" : "Leukemia"
                                  };
-            $super('Cancers', true);
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js]: The rest of the changes don't seem to be related to PT-2825, are they? Can this commit be split in two?

Comment (allasm) [inline components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js]: well, it is kind of related as this enables better "rejection" message. PT-2825 is not a bug, just the message is a bit unclear,and in case of group node there is no message at all. All of these changes are related to improving the messages. I guess a better "fix" is to re-word PT-2825 itself
